docs/rofl/features/proxy.mdx

```diff
@@ -0,0 +1,88 @@
+# ROFL Proxy
+
+The ROFL proxy automatically generates public HTTPS URLs for services in your
+ROFL app. Simply publish a port in your `compose.yaml` and the proxy handles
+TLS certificates and routing.
```

Comment:
Should probably mention that TLS is terminated inside the ROFL app and so there is end-to-end encryption where even the provider doesn't see the traffic.